src/getInfo.spec.js

```diff
@@ -0,0 +1,18 @@
+const getInfo = require('./getInfo');
+
+describe('getInfo method', () => {
+  describe('when invoking the method', () => {
+    const info = getInfo();
+
+    test('should return the correct version', () => {
+      expect(info.version).toBe('1.0.0');
```

Comment:
not a big fan of this, painful everytime you upgrade the version, but ok for now